recipes/pip/bld.bat

```diff
@@ -0,0 +1,9 @@
+python setup.py install
```

Comment:
Also add the arguments `--single-version-externally-managed --record record.txt` here too.